fastapi_limiter/depends.py

```diff
@@ -12,11 +12,11 @@
 class RateLimiter:
     def __init__(
         self,
-        times: conint(ge=0) = 1,
-        milliseconds: conint(ge=-1) = 0,
-        seconds: conint(ge=-1) = 0,
-        minutes: conint(ge=-1) = 0,
-        hours: conint(ge=-1) = 0,
+        times: Annotated[int, Field(ge=0)] = 1,
+        milliseconds: Annotated[int, Field(ge=-1)] = 0,
+        seconds: Annotated[int, Field(ge=-1)] = 0,
+        minutes: Annotated[int, Field(ge=-1)] = 0,
+        hours: Annotated[int, Field(ge=-1)] = 0,
```

Comment:
I migrated according to this
https://github.com/pydantic/pydantic/blob/19fa86cbfcc7b64c5d59c04c38fa80b497f275a2/pydantic/types.py#L157-L182